### PR TITLE
Moving storage connector test method to a util file and use it to test git storage

### PR DESCRIPTION
**Pull request by abrkt**
Trying to unify all tests as mentioned in #150

### CODE REVIEW

File: storageconnector/storageconnectortestutil.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package storageconnector
+
+import (
+	"testing"
+	"regexp"
+	"github.com/stretchr/testify/assert"
+	"github.com/zalando-incubator/pazuzu/shared"
+)
+
+func searchMetaAndFindResultTest(t *testing.T, name string, expected []shared.FeatureMeta, storage StorageReader) {
```
Comment (netme) [inline storageconnector/storageconnectortestutil.go]: It's a good practice to call the Go tests starting with `Test`

File: storageconnector/storageconnectortestutil.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package storageconnector
```
Comment (netme) [inline storageconnector/storageconnectortestutil.go]: The filename should be called `storageconnector_test`

File: storageconnector/storageconnectortestutil.go (hunk 1 above)
Comment (netme) [inline storageconnector/storageconnectortestutil.go]: https://golang.org/pkg/testing/

File: storageconnector/storageconnectortestutil.go (hunk 2 above)
Comment (netme) [inline storageconnector/storageconnectortestutil.go]: The main point behind is to make sure it's only compiled during the `go test`

File: storageconnector/storageconnectortestutil.go (hunk 1 above)
Comment (netme) [inline storageconnector/storageconnectortestutil.go]: Sorry, missed it, these are utility functions, ignore my comment.

File: storageconnector/gitstorage_test.go [hunk 3]
```diff
@@ -4,18 +4,23 @@ import (
...
 	testRepository = filepath.Join("fixtures", "git")
-	storage        StorageReader
+	storage StorageReader
 )
 
 func TestMain(m *testing.M) {
+	spew.Config = spew.ConfigState{
+		DisableCapacities: true,
+		DisablePointerAddresses: true,
+	}
```
Comment (netme) [inline storageconnector/gitstorage_test.go]: Do we really need an extra dependency? Assert library does quite good pretty printing.

Comment (netme) [inline storageconnector/gitstorage_test.go]: Ah, got it. OK.

**Comment by abrkt** [comment]
Changes moved to a feature branch

**Pull request closed without merging**